File: backend/app/schemas/lexical_item.py
# backend/app/schemas/lexical_item.py
from pydantic import BaseModel


class LexicalItemSchema(BaseModel):
    lexical_item_id: int
    category_id: int
    text: str

    class Config:
        orm_mode = True


class LexicalItemListResponse(BaseModel):
    items: list[LexicalItemSchema]

File: android/android/src/main/java/com/wintagma/android/core/model/LexicalItem.kt
package com.wintagma.android.core.model

/**
 * Representa un lexical_item del backend:
 * - lexical_item_id
 * - category_id
 * - text
 *
 * La app puede usarlo en flujos futuros si necesita mostrar listado
 * de items por categoría (/content/items/{category_id}).
 */
data class LexicalItem(
    val lexicalItemId: Int,
    val categoryId: Int,
    val text: String,
)
